doc/api/process.md

```diff
@@ -138,7 +139,9 @@ times nothing happens - but the 10th time, the system becomes corrupted.
 The correct use of `'uncaughtException'` is to perform synchronous cleanup
 of allocated resources (e.g. file descriptors, handles, etc) before shutting
 down the process. It is not safe to resume normal operation after
-`'uncaughtException'`.
+`'uncaughtException'`. To restart a crashed application in a more reliable way,
+whether `uncaughtException` is emitted or not, an external watcher should be
+employed.
```

Comment:
The phrase `an external watcher` is not clear here. Many reading this won't have any idea what that is.